common/reviews/api/rush-lib.api.md

```diff
@@ -395,6 +395,8 @@ export class RushConfiguration {
     tryGetProjectForPath(currentFolderPath: string): RushConfigurationProject | undefined;
     // @beta (undocumented)
     get versionPolicyConfiguration(): VersionPolicyConfiguration;
+    // @beta (undocumented)
+    get versionPolicyConfigurationFile(): string;
```

Comment:
`versionPolicyConfigurationFilePath` to match what we do in other places in Rush

Reply:
Renamed to follow naming conventions